docs/getting-started/explore-intro.asciidoc

```diff
@@ -2,7 +2,7 @@
 
 = Explore
 
-The following section includes an overview of the *Hosts* and *Network* pages.
+The following section includes an overview of the *Hosts*, *Users*, and *Network* pages.
 
 include::network-page-overview.asciidoc[leveloffset=+1]
 include::{security-docs-root}/docs/management/hosts/hosts-overview.asciidoc[leveloffset=+1]
```

Comment:
Consider reordering (both the text and the TOC `include` statements) to match the order in the UI/side nav:
```suggestion
The following section includes an overview of the *Hosts*, *Network*, and *Users* pages.

include::{security-docs-root}/docs/management/hosts/hosts-overview.asciidoc[leveloffset=+1]
include::network-page-overview.asciidoc[leveloffset=+1]
```